docs/spec.md

```diff
@@ -99,7 +99,7 @@ For example:
  - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
 Note that these examples are not universal, in some cases different
-definitions of equivalense for that types might be more appropriate.
+definitions of equivalence for your types might be more appropriate.
```

Comment:
`that` is referring to the types from the examples above. I think we should keep it.

Reply:
Good point.

Reply:
Changed it to "those", just waiting on travis for a second